packages/vm/src/evm/opcodes/functions.ts

```diff
@@ -379,7 +380,7 @@ export const handlers: Map<number, OpHandler> = new Map([
       if (length !== BigInt(0)) {
         data = runState.memory.read(Number(offset), Number(length))
       }
-      const r = bufferToBigInt(keccak256(data))
+      const r = bufferToBigInt(toBuffer(keccak256(data)))
```

Comment:
not sure if we need to initialize a buffer here, can just use `bytesToHex` exported from the `ethereum-cryptography` utils. so `BigInt('0x' + bytesToHex(keccak256(data)))`

also, anywhere you do `toBuffer().toString('hex')` can directly use `bytesToHex`

Reply:
@ryanio Ah, nice, yeah I was wondering if there was a better way to do that. Updated throughout.

However, using `ethereum-cryptography/utils` seems to have introduced a build issue. Getting a strange typescript error during the `npm i` step of all the CI jobs - e.g. `Error: [Node build] Working... ../../node_modules/ethereum-cryptography/utils.d.ts(7,11): error TS2304: Cannot find name 'Crypto'.` (https://github.com/ethereumjs/ethereumjs-monorepo/runs/6106526984?check_suite_focus=true#step:4:102)

`bytesToHex` is just re-exported by `ethereum-cryptography` from `@nobles/hashes`, but we don't have `@nobles/hashes` listed as a dependency anywhere. Maybe it wouldn't hurt to list it as a dependency and use it directly since it's already being installed as a sub-dependency?

Have you seen that error before? I looked into it a bit but ran out of time before needing to go to bed :(

Reply:
I would want to avoid to use `@noble/hashes` directly respectively add as an additional depedency, that's what `ethereum-cryptography` is for to give an (thin layer of) abstraction here.

Reply:
That's interesting because it's crypto with a capital C, which I haven't seen before (these types come from `@types/node` usually). I haven't checked out the code yet to test, but I have a feeling it is related to this type in `ethereum-cryptography/src/utils:L43`: `export const crypto: { node?: any; web?: Crypto }`. I don't have a solution to solve yet, but I can try playing around.

Reply:
Maybe something to forward to Patricio? 🤔

Reply:
It may be because of a different typescript version?

`Crypto` is defined in `typescript/lib/lib.dom.d.ts`.

Reply:
Adding `"dom"` here should fix it

https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/config/tsconfig.json#L15

Reply:
@alcuadrado Thank you - that fixed it :)

@holgerd77 @ryanio Build is working again. Ready for review

Reply:
thanks @alcuadrado!!